share/picongpu/examples/LaserWakefield/lib/python/picongpu/ui_params.py

```diff
@@ -0,0 +1,20 @@
+from eupraxia_picongpu.parameters import *
+
+# if more params are added, make sure to add their names and sensible default
+# values to uiParamDefines.param within picongpu simulation_defines and adapt
+# the *.param files that define those values
```

Comment:
REQUIRED:
grammar define**s**

Reply:
The reference to uiParamDefines.param is outdated. The default defines are to be placed at the beginnings of the respective *.param file.

Reply:
since "define" refers to many files, it is correct. The comment still is outdated as mentioned by @jkelling and will be updated